#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_manpagefix.dpatch by  William Vera <billy@billy.com.mx>
##
## DP: Character cleanup for UTF-8 compatibility.

@DPATCH@
diff -urNad scrot-0.8~/scrot.1 scrot-0.8/scrot.1
--- scrot-0.8~/scrot.1	2001-01-11 11:43:06.000000000 -0600
+++ scrot-0.8/scrot.1	2008-08-15 00:46:36.000000000 -0500
@@ -87,7 +87,7 @@
 \\n prints a newline (ignored when used in the filename)
 .fi
 .SH EXAMPLE
-scrot '%Y-%m-%d_$wx$h.png' -e 'mv $f ~/shots/'
+scrot '%Y\-%m\-%d_$wx$h.png' \-e 'mv $f ~/shots/'
 .br
 This would create a file called something like
 2000-10-30_2560x1024.png and move it to your shots directory.
